server/controllers/middlewares/errorHandle/error.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-unused-vars */
+exports.client = (req, res) => {
+  res.status(404).json('404', { title: 'page not found 404' });
+};
+
+exports.server = (err, req, res, next) => {
```

Comment:
please use more descriptive variable Names
```suggestion
exports.clientError = (req, res) => {
  res.status(404).json('404', { title: 'page not found 404' });
};

exports.serverError = (err, req, res, next) => {
```